home_work/hw_8/luhn_algo/luhn_algo.py

```diff
@@ -0,0 +1,34 @@
+"""
+This module implements the Luhn algorithm for validating credit card numbers.
+"""
+import re
+
+
+class LuhnAlgorithm:
```

Comment:
Do you need to use `class` here? You have used `@staticmethod` for all class methods, staticmethods are used for methods that are not related to class but attached to it.
https://www.programiz.com/python-programming/methods/built-in/staticmethod